Entity/Enemy/Enemy.cpp

```diff
@@ -0,0 +1,18 @@
+//
+// Created by benek on 10/14/24.
+//
+
+#include "Enemy.h"
+#include "EnemyStates.h"
+#include <iostream>
+
+void Enemy::Update() {
+	m_pStateMachine->Update();
+	std::cout << "Enemy Health: " << m_Health << "\n";
```

Comment:
i currently write a lot of stuff to cout. this is not intended to land in prodcution just for easier debugging until we marry the rendering of actual models with the entities and stuff

Reply:
No problem writing to stdout for now. But don't use iostream for that. See tech. design document (orthodox c++). I also use iostreams in the map parser. It is a wrongdoing I did in the past and needs to be fixed.